packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -147,8 +186,25 @@ export default {
   border: 1px solid green;
 }
 
+.c-imageTile-imageContainer {
+  display: block;
+  background-color: rgba($image-tile-background-color, $image-tile-background-opacity);
+  border-radius: $radius-rounded-b;
+  background-image: var(--bg-image);
+  padding-top: (3 / 5) * 100%; // 5:3 aspect ratio
+  position: relative;
+}
+
 .c-imageTile-image {
-    display: block;
+    border-radius: $radius-rounded-b;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
```

Comment:
if this is more for centralised position ignore me - but might be able to drop the setting of the bottom and right values (or top and left)

Reply:
Yeah for sure! We can remove the bottom and right 👍